client/assets/javascripts/components/CommentList.jsx

```diff
@@ -10,12 +10,12 @@ const CommentList = React.createClass({
 
   render() {
     const reversedData = this.props.comments.slice(0).reverse();
-    const commentNodes = reversedData.map((comment, index) => {
+    const commentNodes = this.props.comments.get('comments').map((comment, index) => {
```

Comment:
Need to sort out the reversed data. The reducer is putting the new one at that the top. I'm not sure if we should have the store in reverse order, and not reverse for display or vice versa.